Src/library/vision2/implementation/implementation_interface/properties/ev_positioned_i.e

```diff
@@ -58,6 +58,12 @@ feature -- Measurement
 		deferred
 		end
 
+	dpi: INTEGER
```

Comment:
Why INTEGER and not NATURAL as in EV_MONITOR_DPI_DETECTOR_IMP ?

Reply:
or the reverse


Why NATURAL in EV_MONITOR_DPI_DETECTOR_IMP and not INTEGER ?

Reply:
Done updated to NATURAL.